packages/wallets/src/components/WalletsAddMoreCardBanner/WalletsAddMoreCardBanner.scss

```diff
@@ -12,28 +12,40 @@
         @include mobile {
             width: 16rem;
             height: 9.6rem;
-            padding: 0.8rem;
+            padding: 1rem;
             flex-shrink: 0;
         }
         &-header {
             display: flex;
             justify-content: space-between;
         }
         &-button {
+            background-color: #ffffff;
+            border: none;
+            border-radius: 5px;
             cursor: pointer;
+            display: flex;
+            align-items: center;
+            justify-content: center;
             width: 100%;
-            padding: 0.6rem 1.6rem;
+            padding: 1rem;
             font-weight: 700;
 
             @include mobile {
                 padding: 0.3rem 0.8rem;
             }
+
+            &-icon {
+                margin-right: 0.5rem;
+            }
+
             &--is-added {
                 opacity: 0.3;
                 cursor: default;
             }
         }
         &-landing-company {
+            align-self: flex-start;
```

Comment:
This prevents the landing company badge to stretch its height to fit the flexbox due to the SVG scaling